workspace/read-manifest/src/index.ts

```diff
@@ -30,11 +27,10 @@ async function readManifestRaw (dir: string): Promise<unknown> {
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function validateWorkspaceManifest (manifest: any): manifest is WorkspaceManifest | undefined {
+function validateWorkspaceManifest (manifest: unknown): asserts manifest is WorkspaceManifest | undefined {
```

Comment:
TypeScript has 2 kinds of [user-defined type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards).

1. Predicates that simply return `true` or `false` without throwing.
2. [Assertion functions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions) that return void and throw if the assertion fails.

This function is currently written as an in-between. I think the intention was for this to be an _Assertion function_, which avoids us having to `return true` and an `if` statement in the calling path.